src/universe/qual/Lost.java

```diff
@@ -11,13 +11,12 @@
 
 /**
  * Ownership information is not expressible from the current viewpoint.
- * Can be used explicitly if allowLost is true. Otherwise forbidden.
```

Comment:
Keep this comment and with the option document that it currently doesn't work.

Reply:
Does this comment look good?